packages/fxa-content-server/app/scripts/views/behaviors/settings.js

```diff
@@ -23,8 +23,12 @@ export default function(defaultBehavior, options = {}) {
   const behavior = function(view, account) {
     return account.isSignedIn().then(isSignedIn => {
       if (isSignedIn) {
-        let success = t('Account verified successfully');
         let endpoint = 'settings';
+        let success;
+
+        if (view.type === 'recovery_key') {
```

Comment:
Ideally, we would be able to pass in the `options.success` from when we navigate to the page with the [invokeBrokerMethod](https://github.com/mozilla/fxa/blob/78cbb8cd5bee464262a965b64213a5f2d7cf1a1e/packages/fxa-content-server/app/scripts/views/post_verify/verified.js#L57). However, we would have to add a few of them and that might be more complexity than it is worth 🤷🏽‍♂️